build/android-build.py

```diff
@@ -1,120 +1,100 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
It will work.

`/usr/bin/env python` will find the python interpreter that is on your `PATH`.
On your machine it will find `/usr/bin/python`, unless you have installed another version in i.e. `/usr/local/bin`.

http://stackoverflow.com/a/2429517/104451